src/include/duckdb/storage/serialization/parsed_expression.json

```diff
@@ -258,6 +258,11 @@
         "id": 201,
         "name": "expr",
         "type": "ParsedExpression*"
+      },
+      {
+        "id": 202,
+        "name": "deprecated",
```

Comment:
I do agree that we likely don't want this, if the argument is that we want to test old vs new syntax, I think we can do so without forwarding this information to the serialized level